<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgMain</class>
 <widget class="QDialog" name="dlgMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="base_BG" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f0f1f2;
border-radius: 30px;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spoqa Han Sans Neo Bold</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Calculator</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>317</x>
      <y>50</y>
      <width>55</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Spoqa Han Sans Neo Midium</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    color: #424242;
    font: 12px &quot;Spoqa Han Sans Neo Midium&quot;;
}
</string>
    </property>
    <property name="text">
     <string>by momo</string>
    </property>
   </widget>
   <widget class="QWidget" name="display_BG" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>336</width>
      <height>126</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: white;
	border-radius: 14px;
	box-shadow: 10px 10px 10px rgba(196, 203, 188, 1.0);
}</string>
    </property>
    <widget class="QWidget" name="display" native="true">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>312</width>
       <height>102</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #c4cbbc;
	border-radius: 14px;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);
}</string>
     </property>
     <widget class="QLineEdit" name="txtInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>270</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
    color: #424242;
    font: 12px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPostfix">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>270</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
    color: #424242;
    font: 12px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtResult">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>270</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
    color: #424242;
    font: 30px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnAllClear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #ee8218;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivide">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultiply">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLeftParen">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinus">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSeven">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNine">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRightParen">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEight">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFour">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEqual">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>60</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #41758a;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #ffffff;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFive">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSix">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThree">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>460</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOne">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTwo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZero">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>460</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #f7f7f7;
	border-radius: 16px;
	border: 1px solid #dddedf;
	box-shadow: 10px 10px 5px rgba(196, 203, 188, 1.0);

    color: #424242;
    font: 24px &quot;Spoqa Han Sans Neo Bold&quot;;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
